scripts/setup.js

```diff
@@ -29,12 +28,16 @@ const DOWNLOAD_HISTORY_MESSAGE = `Download the entire history of terms of servic
     }]);
 
     if (answer.history === NEW_HISTORY_MESSAGE) {
-      await fs.mkdir(config.get('history.dataPath'), { recursive: true });
-      const git = simpleGit(path.resolve(__dirname, '../', config.get('history.dataPath')));
+      await fs.mkdir(config.get('history.snapshotsPath'), { recursive: true });
+      await fs.mkdir(config.get('history.versionsPath'), { recursive: true });
+      let git = simpleGit(path.resolve(__dirname, '../', config.get('history.snapshotsPath')));
+      await git.init();
+      git = simpleGit(path.resolve(__dirname, '../', config.get('history.versionsPath')));
```

Comment:
> It would be more efficient not to wait each time, but to instead chain promises so that initializing the first repo doesn't wait upon the creation of the first directory.